pkg/types/validation/installconfig.go

```diff
@@ -1050,6 +1050,12 @@ func validateFeatureSet(c *types.InstallConfig) field.ErrorList {
 			allErrs = append(allErrs, field.Forbidden(field.NewPath("platform", "azure", "userTags"), errMsg))
 		}
 
+		if c.AWS != nil {
+			if len(c.AWS.HostedZoneRole) > 0 {
+				allErrs = append(allErrs, field.Forbidden(field.NewPath("platform", "aws", "hostedZoneRole"), errMsg))
```

Comment:
Like GCP is this going to cause any issues with the install when using the out of tree provider?

Reply:
I believe out-of-tree is actually coming out of tech preview but no issues so far